app/src/main/java/org/oppia/app/help/FAQ/FAQActivityPresenter.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.help.FAQ
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import javax.inject.Inject
+
+/** The presenter for [FAQActivity]. */
+@ActivityScope
+class FAQActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.faq_activity)
+    activity.supportFragmentManager.beginTransaction().add(
```

Comment:
Check other files regarding how to layout fragment, in this implementation there are chances that we will make multiple copies of `FAQFragment`